sql/variable.md

```diff
@@ -32,6 +32,70 @@ SET @@autocommit = 1;
 
 * `LOCAL` 以及 `@@local.` 是 `SESSION` 以及 `@@session.` 的同义词
 
+### 系统变量的作用机制
+
+* 会话范围的系统变量仅仅会在创建会话时才会根据全局范围系统变量初始化自己的值。更改全局范围的系统变量不会改变已经创建的会话正在使用的系统变量的值。
+
+```sql
```

Comment:
Please indent this code block with 4 spaces.

Reply:
In this variable.md file, there are two other code blocks that need to be indented with 4 spaces, plz indent them as well.

Reply:
PTAL @yikeke